src/components/button/mini-button/mini-button.scss

```diff
@@ -0,0 +1,15 @@
+@use '../../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  display: flex;
```

Comment:
Was it flex before as well? just wondering because a native button would have inline-block, and a mini button is kind of a mini thing which probably should not have block behavior